taca/analysis/analysis.py

```diff
@@ -1,237 +1,219 @@
-                             "demultiplexing for run {}".format(run.id)))
+                # For now MiSeq specific case. Process only NGI-run, skip all the others (PhD student runs)
+                logger.warn("sRun {} marked as {}, "
+                            "TACA will skip this and move the run to "
+                            "no-sync directory".format(run.id, run.get_run_type()))
+                if 'storage' in CONFIG:
+                    run.archive_run(CONFIG['storage']['archive_dirs'][run.sequencer_type])
+                return 
+            # Otherwise it is fine, process it
+            logger.info(("Starting BCL to FASTQ conversion and demultiplexing for run {}".format(run.id)))
             run.demultiplex_run()
-
         elif run.get_run_status() == 'IN_PROGRESS':
             logger.info(("BCL conversion and demultiplexing process in "
-                             "progress for run {}, skipping it"
-                             .format(run.id)))
-            #in the case of Xten retruns, in future have a look to Cycles.txt
-            #in the case of HiSeq check that partial demux are done and performs aggregation if this is the case
+                         "progress for run {}, skipping it".format(run.id)))
+            # In the case of Xten returns, in future have a look to Cycles.txt
+            # in the case of HiSeq check that partial demux are done and performs aggregation if this is the case
             run.check_run_status()
 
-        # previous elif might change the status to COMPLETED (in HiSeq), therefore to avoid skip
-        # a cicle take the last if out of the elif
+        # previous elif might change the status to COMPLETED (in HiSeq), therefore to avoid skipping
+        # a cycle take the last if out of the elif
         if run.get_run_status() == 'COMPLETED':
-            logger.info(("Preprocessing of run {} is finished, "
-                             "tranferring it".format(run.id)))
-            #in the case of of HiSeq this function computes undetermined indexes for NoIndex lanes
+            logger.info(("Preprocessing of run {} is finished, transferring it".format(run.id)))
+            # In the case of of HiSeq this function computes undetermined indexes for NoIndex lanes
             if not run.compute_undetermined():
-                return None
-            #otherwise I can procced to QC
-            #check the run QC
+                return
+            # Otherwise I can proceed to QC
+            # Check the run QC
             run_QC_status = run.check_QC()
-            #store QC results in appropriate file and mail user if failed
-            qc_file = os.path.join(CONFIG['analysis']['status_dir'], 'qc.tsv')
-            #this method is implemented in Runs
-            run.post_qc(qc_file, run_QC_status, log_file=CONFIG['log']['file'], rcp=CONFIG['mail']['recipients'])
-            #upload to statusDB
+            if run_QC_status:
```

Comment:
@jfnavarro this change affects our workflow... in this way if run_QC_status is FALSE we do not send the mail, which is not the correct behaviour